Dockerfile

```diff
@@ -1,45 +1,58 @@
-FROM node:16.16.0-alpine3.15 AS base
-
 # Install dependencies only when needed
-FROM base AS deps
+FROM node:16-alpine AS deps
```

Comment:
We would still like to have the version pinned to a specific node version since that gives us consistency.